packages/babel-core/src/api/node.js

```diff
@@ -30,6 +30,195 @@ export function Plugin(alias) {
   throw new Error(`The (${alias}) Babel 5 plugin is being run with Babel 6.`);
 }
 
+// Sorry about all the MBs...
+/* eslint-disable max-len */
+export const guy = [
```

Comment:
Hi, it's Babel Bulbasaur from SanFran Recruiting office. We saw this incredible Pull Request and want to offer you a full time job as a open source trainer. The job mainly consists of playing Pokemon and telling people that this practise is a good for having a strong codebase, please don't hesitate to contact us at: dontclosethisthread@sanfranrecruit.us